piikii-output-web/tmap/src/main/kotlin/com/piikii/output/web/tmap/adapter/TmapNavigationAdapter.kt

```diff
@@ -50,9 +70,16 @@ class TmapNavigationAdapter(
             .retrieve()
             .body<TmapRouteInfoResponse>()
             ?.toDistance()
-            ?: throw PiikiiException(
-                exceptionCode = ExceptionCode.ROUTE_PROCESS_ERROR,
-                detailMessage = "fail to request tmap api call, start(${start.x}, ${start.y}), end(${end.x}, ${end.y})",
-            )
+            ?: getEmptyDistanceOfError(start, end)
+    }
+
+    private fun getEmptyDistanceOfError(
```

Comment:
사소하지만, 이런 네이밍이 좋을 것 같아요.
```suggestion
    private fun getEmptyDistanceWithError(
```